coalib/parsing/LineParser.py

```diff
@@ -55,6 +56,19 @@ def parse(self, line):
         line, comment = self.__separate_by_first_occurrence(
             line,
             self.comment_separators)
+
+        # Comment starting with # should have whitespace after #
+        # Example: C# is the hybrid of C and C++.
+        if '#' in line:
+            start_index = line.index('#')
+            if start_index == 0 and line[1] != ' ':
+                logging.warning('The comment line does not have \
+                                 whitespace after #')
+            elif line[start_index - 1] != ' ' \
```

Comment:
@Nosferatul you want that I should remove the condition `start_index == 0`. Am I thinking correct?
But I mentioned because when comment starts with # then it is 0, then i have to check that there is whitespace after # or not.